gocron.go

```diff
@@ -8,6 +8,7 @@ package gocron
 
 import (
 	"errors"
+	"fmt"
```

Comment:
You should not really print anything on console for the package

Reply:
From what I can see the fmt package here is used just to wrap errors, so this should be fine :-)